inst/qml/designOfExperiments.qml

```diff
@@ -34,15 +34,43 @@ Form
 			{ value: "factorial", label: qsTr("Factorial") 			},
 			{ value: "screening", label: qsTr("Screening") 			},
 			{ value: "response",  label: qsTr("Response surface") 	},
-			{ value: "mixture",   label: qsTr("Mixture") 			},
-			{ value: "taguchi",   label: qsTr("Taguchi")			}
+			{ value: "mixture",   label: qsTr("Mixture") 			}
 		]
 	}
 
 	CheckBox
 	{
 		name: 									"displayDesign"
-		label:									qsTr("Preview design")
+		label:									"Preview design"
+	}
+
+	GroupBox
+	{
+		title: 									qsTr("Factor Info")
+		name:									"factorInfo"
```

Comment:
You don't necessarily have to give a `name` to a `GroupBox` because it is not an option that is passed to R